rules/crowdstrike_rules/event_stream_rules/crowdstrike_new_user_created.yml

```diff
@@ -0,0 +1,89 @@
+AnalysisType: rule
+Filename: crowdstrike_new_user_created.py
+RuleID: "Crowdstrike.NewUserCreated"
+DisplayName: "Crowdstrike New User Created"
+Reports:
+  MITRE ATT&CK:
+    - T1136.003 # Create Cloud Account
+CreateAlert: false
```

Comment:
Is there a way to tell if the new user has admin privileges?  Maybe a CR with user created followed by admin role assigned

Reply:
Not without caching. Since the information on what user was created (and had roles assigned to) is in the `AuditKeys` list, we can't use it for match criteria. The role assignment does appear as a separate alert though

Reply:
Oh lol nvm, I forgot about `alert_context`